app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -30,7 +30,8 @@ public static SessionStore get() {
         }
         return mInstance;
     }
-    public static final String DEFAULT_URL = "resource://android/assets/html/index.html";
+    public static final String HOME_URL = "https://webxr.today/?region=";
+    public static String DEFAULT_URL = "https://webxr.today";
```

Comment:
I would prefer something like this:
```java
private static final String HOME_URL_COUNTRY_CODE_TEMPLATE = "https://webxr.today/?region=";
private static final String DEFAULT_HOME_URL = ""https://webxr.today";
private static String sHomeUrl = DEFAULT_HOME_URL;

public static void setHomeUrlContryCode(final String aCountryCode) {
    if (aCountryCode == null) {
        sHomeUrl = DEFAULT_URL;
        return;
    }
    sHomeUrl = HOME_URL_COUNTRY_CODE_TEMPLATE + aCountryCode;
}

public static String getHomeUrl() {
    return sHomeUrl;
}
```

And then replace `SesionStore.DEFAULT_URL` with `SesionStore.getHomeUrl()`